src/common/util/parse-aspect-ratio.ts

```diff
@@ -1,24 +1,30 @@
-export default function parseAspectRatio(input) {
-  // Handle 16x9, 16:9, 1.78x1, 1.78:1, 1.78
-  // Ignore everything else
-  function parseOrThrow(num) {
-    const parsed = parseFloat(num);
-    if (isNaN(parsed)) {
-      throw new Error(`${num} is not a number`);
-    }
-    return parsed;
+// Handle 16x9, 16:9, 1.78x1, 1.78:1, 1.78
+// Ignore everything else
+const parseOrThrow = (num) => {
+  const parsed = parseFloat(num);
+  if (isNaN(parsed)) {
+    throw new Error(`${num} is not a number`);
+  }
+  return parsed;
+};
+
+export default function parseAspectRatio(input: string) {
+  if (!input) {
+    return null;
   }
   try {
-    if (input) {
-      const arr = input.replace(":", "x").split("x");
-      if (arr.length === 0) {
-        return null;
-      }
+    if (input.substr(input.length - 1) === "%") {
```

Comment:
`if (input.startsWith("%")) {`

Reply:
Oh that's a thing https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

Reply:
`endsWith` that would be.

Reply:
Of course. Stupid mistake. Editing my comment so noone will ever know.